src/style/_base.scss

```diff
@@ -0,0 +1,10 @@
+input {
```

Comment:
Setting default (16px) here as we are setting root font-size to 62.5% - `1rem => 10px`